### DISCUSSION TITLE
configuration won't save, issue with stations

### DISCUSSION THREAD

**Discussion by sw41** [fuatakgun/eufy_security#1144] [General]
Error: `Failed to save add-on configuration, Invalid list for option 'stations' in eufy-security-ws (402f1039_eufy_security_ws).`

I am trying to configure the add-on eufy-security-ws into home assistant and have it downloaded but the configuration is asking me for "stations" and I do not know what to put here.  Tried the IP of my HA and of the camera but it does not like either of those. 

I only have 2 of the indoor CamMini and no cloud storage.

**Comment by AngryUndeadDad** [comment]
I left mine blank.

**Comment by xCephandrius** [comment]
In case you're still running into this, or anyone else is searching for answer as I was: you need to format it correctly. See below:
I included SN and IP of Homebase
Make sure you indent the second line or it will continue to fail.

```
- serial_number: T8030XXXXXXXXX
  ip_address: 10.10.10.10
```

**Reply by regetskcob** [reply]
Now getting another error: "does not match regular expression ^(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$." do I need to wrap it in kind of a list? how should the content look like?
